index.html

```diff
@@ -188,7 +188,7 @@ <h2>Multy-Post Stories</h2>
               /></a>
             </li>
           </ul>
-          <button class="project-button">See Project</button>
+          <button class="project-button projButtFour">See Project</button>
```

Comment:
- [ ] Your popup must follow the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins,...)  kindly make this requested changes: 


- Please blur the sub background when the popup is open, it will make the popup more like Figma 👍🏿
- It would be better if the background is not scrollable when the popup page is open. you can use the `overflow:hidden;` CSS property to avoid this. 😄 
- Kindly remove the extra padding-top and bottom of your popup text tittle `Multy-Post Stories` 
- Make sure the max width of your popup windows image takes the full width as shown on the Figma design
- Let's add the missing icons to the  `See live and See source` button ✍🏻 

Your Implementation | Figma Design
:---:|:---:
![18](https://user-images.githubusercontent.com/79058364/195240439-35fbd2b5-d128-4cad-a0a2-029fe44a7945.png)|![Screenshot from 2022-10-12 05-00-43](https://user-images.githubusercontent.com/79058364/195239977-37fb4625-af99-43e0-8509-0edd240c0d50.png)
 


- [ ]  Please do the same for the desktop size 👨🏻‍🎨 🎨 

- Actually, I don't see the full height of your page popups when I'm on the desktop size, please make sure 100% height is shown 👨🏻‍🎨

Your Implementation | Figma Design
:---:|:---:
![18](https://user-images.githubusercontent.com/79058364/195241361-058717ce-7d62-4c4b-b52d-b7c422f488c5.png) | ![Screenshot from 2022-10-12 05-13-25](https://user-images.githubusercontent.com/79058364/195241407-f696f0f5-78d1-4853-b3ea-baeebab49b3d.png)

Reply:
Hi, that's the mobile version in a bigger resolution. Since the small cards have display none on bigger resolutions (768+px) the design looks like this: 
![desktop version](https://user-images.githubusercontent.com/111214189/195247652-ed3fcbe8-2242-49dd-83c3-117db57310d4.png)